src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -450,14 +450,27 @@ private async Task RefreshAsync(string key, DateTimeOffset? absExpr, TimeSpan? s
 
         /// <inheritdoc />
         public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
This class doesn't implement a finalizer anyway. I'm not sure this change is necessary.

Reply:
@Pilchie Since this is public and not sealed, wouldn't this be useful if derived types have a finalizer?

(I'm not sure, just asking)

Reply:
When following the full `IDisposable` pattern from the framework design guidelines, the call to `GC.SuppressFinalize()` is always required. See https://github.com/dotnet/roslyn-analyzers/issues/4745.